nautobot/extras/models/mixins.py

```diff
@@ -18,6 +18,14 @@ class Meta:
 
     is_approval_workflow_model = True
 
+    # Reverse relation so that deleting a ApprovableModelMixin automatically deletes any approval workflow instances related to it.
+    associated_approval_workflow_instances = GenericRelation(
+        "extras.ApprovalWorkflowInstance",
+        content_type_field="object_under_review_content_type",
+        object_id_field="object_under_review_object_id",
+        related_query_name="associated_approval_workflow_instances_%(app_label)s_%(class)s",  # e.g. 'associated_object_approval_workflow_instances_dcim_device'
```

Comment:
I'm a bit worried about the length of this name - any ideas for a shorter alternative?